### PR TITLE
Updates to github org/repo names

### LINKED ISSUE #32: Update to use Adoptium references and links
Now that this repo and temurin-build is transitioned over, we should update to use the new names/locations for build scripts.

### PR DESCRIPTION
**Pull request by smlambert**
Fixes #32 

Signed-off-by: Shelley Lambert <slambert@gmail.com>

### CODE REVIEW

**Comment by smlambert** [comment]
Noting that I left some reference to AdoptOpenJDK as this action needs to go and download some utilities to set up environment properly and those are currently hosted on the existing Jenkins server, and a bootJDK from the old API for jdk10 releases (which will not be served from new API, I presume).  If I update those references, this github action will be broken.

File: SECURITY.md [hunk 1]
```diff
@@ -12,6 +12,6 @@ We currently support the following versions with security updates.
 
 ## Reporting a Vulnerability
 
-Please report vulnerabilities to [security@adoptopenjdk.net](security@adoptopenjdk.net).
+Please report vulnerabilities to [security@adoptium.net](security@adoptium.net).
```
Comment (karianna) [inline SECURITY.md:15]: I changed this to be TBD in the other repo as its not set up yet

**Review by karianna — CHANGES_REQUESTED** [review]
nitpick

**Comment by karianna** [comment]
> Noting that I left some reference to AdoptOpenJDK as this action needs to go and download some utilities to set up environment properly and those are currently hosted on the existing Jenkins server, and a bootJDK from the old API for jdk10 releases (which will not be served from new API, I presume). If I update those references, this github action will be broken.

Can we cache those things locally / on a different host?  We could in theory even host on adoptium.net website...

**Comment by smlambert** [comment]
I presume we will build them on the 'new Jenkins server' with its new domain name, at which point we can do a 2nd PR in this repo.  But we can discuss in PMC meeting.

**Review by sophia-guo — APPROVED** [review]

**Review by karianna — APPROVED** [review]

**Pull request merged**